packages/teacher-app/public/locales/en/translation.json

```diff
@@ -283,5 +283,11 @@
   "ENTER_WORKSHEET_DETAILS": "Enter Worksheet Details",
   "SHOW_QUESTIONS_BASED_ON": "Show questions based on",
   "PLEASE_SELECT_LIMIT": "Please select limit",
-  "QUESTION_NOT_FOUND": "No question found for this filter"
+  "QUESTION_NOT_FOUND": "No question found for this filter",
+  "NO_STUDENT_HAS_ACHIEVED_ATTENDANCE_THIS_WEEK": "No Student Has Achieved 100% Attendance This Week",
```

Comment:
pls review the message style. should it be like "No student has achieved ..."

Reply:
This is review @RinkleDang  and change the message. she have access to change this file.